src/coord/src/catalog/builtin.rs

```diff
@@ -1361,6 +1401,149 @@ pub const PG_SETTINGS: BuiltinView = BuiltinView {
+    sql: "CREATE VIEW mz_scheduling_parks AS SELECT
+    worker, slept_for, requested, pg_catalog.count(*) AS count
+FROM
+    mz_catalog.mz_scheduling_parks_internal
+GROUP BY
+    worker, slept_for, requested",
+    id: GlobalId::System(5029),
+    needs_logs: true,
+};
+
+pub const MZ_MESSAGE_COUNTS: BuiltinView = BuiltinView {
+    name: "mz_message_counts",
+    schema: MZ_CATALOG_SCHEMA,
+    sql: "CREATE VIEW mz_message_counts AS
+WITH sent_cte AS (
+    SELECT
+        channel,
+        source_worker,
+        target_worker,
+        pg_catalog.count(*) AS sent
+    FROM
+        mz_catalog.mz_message_counts_sent_internal
+    GROUP BY
+        channel, source_worker, target_worker
+),
+received_cte AS (
+    SELECT
+        channel,
+        source_worker,
+        target_worker,
+        pg_catalog.count(*) AS received
+    FROM
+        mz_catalog.mz_message_counts_received_internal
+    GROUP BY
+        channel, source_worker, target_worker
+)
+SELECT
+    sent_cte.channel,
+    sent_cte.source_worker,
+    sent_cte.target_worker,
+    sent_cte.sent,
+    received_cte.received
+FROM sent_cte JOIN received_cte USING (channel, source_worker, target_worker)",
+    id: GlobalId::System(5030),
+    needs_logs: true,
+};
+
+pub const MZ_DATAFLOW_OPERATOR_REACHABILITY: BuiltinView = BuiltinView {
+    name: "mz_dataflow_operator_reachability",
+    schema: MZ_CATALOG_SCHEMA,
+    sql: "CREATE VIEW mz_dataflow_operator_reachability AS SELECT
+    address,
+    port,
+    worker,
+    update_type,
+    timestamp,
+    pg_catalog.count(*) as count
+FROM
+    mz_catalog.mz_dataflow_operator_reachability_internal
+GROUP BY address, port, worker, update_type, timestamp",
+    id: GlobalId::System(5031),
+    needs_logs: true,
+};
+
+pub const MZ_ARRANGEMENT_SIZES: BuiltinView = BuiltinView {
+    name: "mz_arrangement_sizes",
+    schema: MZ_CATALOG_SCHEMA,
+    sql: "CREATE VIEW mz_arrangement_sizes AS
+WITH batches_cte AS (
+    SELECT
+        operator,
+        worker,
+        pg_catalog.count(*) AS batches
+    FROM
+        mz_catalog.mz_arrangement_batches_internal
+    GROUP BY
+        operator, worker
+),
+records_cte AS (
+    SELECT
+        operator,
+        worker,
+        pg_catalog.count(*) AS records
+    FROM
+        mz_catalog.mz_arrangement_records_internal
+    GROUP BY
+        operator, worker
+)
+SELECT
+    batches_cte.operator,
+    batches_cte.worker,
+    records_cte.records,
+    batches_cte.batches
+FROM batches_cte JOIN records_cte USING (operator, worker)",
+    id: GlobalId::System(5032),
+    needs_logs: true,
+};
+
+pub const MZ_ARRANGEMENT_SHARING: BuiltinView = BuiltinView {
+    name: "mz_arrangement_sharing",
+    schema: MZ_CATALOG_SCHEMA,
+    sql: "CREATE VIEW mz_arrangement_sharing AS
+SELECT
+    operator,
+    worker,
+    pg_catalog.count(*) AS count
+FROM mz_catalog.mz_arrangement_sharing_internal
+GROUP BY operator, worker",
+    id: GlobalId::System(5033),
+    needs_logs: true,
+};
```

Comment:
I hadn't caught that count view was missing in the earlier commit. Sorry!

What do you think about extending `logging.td` with queries for these?

Reply:
Just to be clear: the current commit LGTM. I'm mainly asking for the tests 😄

I think we had a related case in https://github.com/MaterializeInc/materialize/pull/6445#pullrequestreview-634867678.

Reply:
Added a test to the PR.